问题1、libhdf5_hl.so.10:cannot open shared object file:No such file or directory.
方法：1、使用find命令能找到libhdf5_hl.so.10文件。并且已经把路径加到PATH和LD_LIBRARY_PATH环境变量中。
2、在命令行中执行相同的语句工作正常，但是在jupyter notebook交互式中执行报上述错误。
3、在激活虚拟环境（source activate py27）后，添加环境变量：
> export LD_LIBRARY_PATH=/home/zhangkai/anaconda3/envs/py27/lib:$LD_LIBRARY_PATH

然后再打开jupyter notebook，一切正常。